src/surfaces/actionBar/actionBarActionsButtonDrawerOption.jsx

```diff
@@ -45,23 +50,24 @@ class ActionBarActionsButtonDrawerOption extends React.PureComponent {
         } = this.props;
 
         if (_.isFunction(option.onClick)) {
-            if (option.disabled) {
-                return false;
+            if (option.disable) {
+                return null;
             }
 
-            if (option.requiresPrompt) {
-                if (_.isFunction(onRequestPrompt)) {
-                    onRequestPrompt(option);
-                    return false;
-                }
+            if (option.requiresPrompt && _.isFunction(onRequestPrompt)) {
+                onRequestPrompt(option);
+
+                return null;
             }
 
             option.onClick();
-            return false;
+
+            return null;
         }
 
         onClick(isSelected ? {} : option);
-        return true;
+
+        return null;
```

Comment:
I think we can just do early `return;` as necessary and elide final `return null;`.  I don't think we have to return a value.

I think the only exception to that is when rendering: we should _always_ return _something_ from rendering functions, i.e. `null` if we do in fact want to render nothing.  But for event handling functions and such, I think it is okay if they have void return type.  If your opinion differs, we can discuss.

Reply:
Fixed.

Reply:
Looks good now (void returns).  Resolving.